person.rb

```diff
@@ -18,11 +19,17 @@ def of_age?
     @age >= 18
   end
 
+  public
+
   def can_use_services?
     of_age? || @parent_permission
   end
 
   def correct_name
     @name
   end
+
+  def add_rental(rental)
+    @rentals << rental
+  end
```

Comment:
- [ ] The person itself should be sent as a parameter to create the new rental, achieving the 'has-many' association. It should look something similar to this 

```
 def add_rental(person, date)
    Rental.new(date, book, self)
  end
  
```